docs/src/pages/demos/tables/CustomPaginationActionsTable.js

```diff
@@ -118,6 +118,7 @@ function CustomPaginationActionsTable() {
       createData('Oreo', 437, 18.0),
     ].sort((a, b) => (a.calories < b.calories ? -1 : 1)),
   );
+
```

Comment:
this was auto generated by the `yarn docs:typescript:formatted` even though the `TS` version does not have these spaces

Reply:
That's fine. This is a transpilation artifact. The whitespace is inserted because the types added additional whitespace in the TS version and `babel` tries to align the corresponding lines. Since JS naturally has less lines whitespace gets inserted